src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should discard the remainder', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+  });
+
+  test('should return [0, 0] for catAge = 0, dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for catAge = 14, dogAge = 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [1, 1] for catAge = 15, dogAge = 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  test('should return [1, 1] for catAge = 23, dogAge = 23', () => {
+    expect(getHumanAge(23, 23)).toEqual([1, 1]);
+  });
+
+  test('should return [2, 2] for catAge = 24, dogAge = 24', () => {
```

Comment:
The test case doesn't seem to be correct. If catAge = 24 and dogAge = 24, the conversion to human years shouldn't be [2, 2]. Please confirm with the conversion rules.